File: 1. Begginer/7- Pida al usuario dos números enteros (que se guardarán en las variables a y b) y muestre su suma en pantalla/main.go
package main

import "fmt"

func main() {
	var a int
	var b int
	fmt.Println("Ingrese el primer número: ")
	fmt.Scanf("%d", &a)
	// %d in go is for integer and also user & to indicate address of variable in memory like C
	fmt.Println("Ingrese el segundo número: ")
	fmt.Scanf("%d", &b)
	fmt.Println("El resultado es", a + b)
}

File: 1. Begginer/7- Pida al usuario dos números enteros (que se guardarán en las variables a y b) y muestre su suma en pantalla/main.c
#include<stdio.h>

int main()
{
	int a;
	int b;
	printf("%s", "Ingrese un número: ");
	scanf("%i", &a); /* & is to indicate address in memory for variable, in this case the input of user */
	printf("%s", "Ingrese otro: ");
	scanf("%i", &b);
	printf("El resultado es %i", a + b);
}